xml/selinux.xml

```diff
@@ -417,17 +341,83 @@ system_u:object_r:var_t var</screen>
     </imageobject>
    </mediaobject>
   </figure>
-
+  
   <para>
-   After installing the &selnx; packages, you need to modify the &grub; boot
-   loader. Do this from &yast;, select <menuchoice>
-   <guimenu>System</guimenu> <guimenu>Boot Loader</guimenu> <guimenu>Kernel
-   Parameters</guimenu></menuchoice>. Now add the following parameters to
-   the <guimenu>Optional Kernel Command Line Parameters</guimenu>:
+    This does not install a reference policy. See 
+    <xref linkend="sec-selinux-getpolicy"/> for information on
+    installing an &opensuse; reference policy for testing. 
```

Comment:
-reference. I won't add comments for this anymore, please remove it everywhere

Reply:
:+1: